### PR TITLE
cache docs examples

### PR DESCRIPTION
**Pull request by lheagy**
- get example cache pathed in to the correct location. 
Before:
image

After: 
image

- I will squash when we merge this (most of the commits are me messing with travis).

### CODE REVIEW

**Comment by rowanc1** [comment]
Does this download and then re-upload the builds?

File: .travis.yml [hunk 1]
```diff
@@ -64,20 +64,25 @@ install:
       openssl aes-256-cbc -K $encrypted_7e0a8632ac3f_key -iv $encrypted_7e0a8632ac3f_iv -in credentials.tar.gz.enc -out credentials.tar.gz -d ;
       tar -xzf credentials.tar.gz ;
       gcloud auth activate-service-account --key-file credentials/client-secret-docs.json ;
-      gsutil cp gs://simpeg/_build.tar.gz $PWD/docs/ ;
-      mkdir $PWD/docs/_build ;
-      tar -xzf $PWD/docs/_build.tar.gz -C $PWD/docs/_build ;
-      ls $PWD/docs/_build/ ;
+      gsutil cp gs://simpeg/_build.tar.gz $PWD/docs ;
+      gsutil cp gs://simpeg/examples.tar.gz $PWD/docs/content ;
```
Comment (lheagy) [inline .travis.yml]: downloads

File: .travis.yml [hunk 2]
```diff
@@ -102,7 +107,10 @@ after_success:
         gcloud auth activate-service-account --key-file credentials/client-secret.json ;
         export GAE_PROJECT=simpegdocs ;
       elif [ "$TRAVIS_BRANCH" = "$DEV_BRANCH" ]; then
+        cd docs; tar -cvzf _build.tar.gz _build ;
+        cd content; tar -cvzf examples.tar.gz examples ; cd ..;
         gsutil cp docs/_build.tar.gz gs://simpeg ;
+        gsutil cp docs/content/examples.tar.gz gs://simpeg ;
```
Comment (lheagy) [inline .travis.yml]: re-uploads only on dev

Comment (lheagy) [inline .travis.yml]: (new examples will continue to be run and tested until they get to dev)

File: .travis.yml [hunk 3]
```diff
@@ -64,20 +64,25 @@ install:
...
+      tar -xzf examples.tar.gz examples;
+      cd ../../ ;
     fi
 
 # Run test
 script:
   # - nosetests $TEST_DIR --with-cov --cov SimPEG --cov-config .coveragerc -v -s
   - if [ "$TEST_DIR" = "$DEPLOY_DIR" -a ${TRAVIS_PYTHON_VERSION:0:1} == "3" ]; then
-      cd docs; make html-noplot ; cd ../;
+      cd docs; make html-noplot ; cd ../ ;
```
Comment (lheagy) [inline .travis.yml]: on python 3, we check that the docs compile (no nitpick and no testing of examples)

**Pull request merged**